components/ui/nextButton.tsx

```diff
@@ -1,31 +1,14 @@
-import { Button } from "./button";
+import { useTranslations } from "next-intl";
 
-const NextButton = () => {
+const NextButton = (props: { trigger: boolean }) => {
+  const t = useTranslations("NextButton");
   return (
-    <div className="fixed left-1/2 top-3/4 -translate-x-1/2 transform">
-      <Button className="relative" variant="next" size="xs">
-        <p className="fixed left-1/2 -translate-x-1/2 transform text-base font-semibold text-[#6C1F1F]">
-          ถัดไป
-        </p>
-        <div className="absolute right-1">
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="20"
-            height="20"
-            viewBox="0 0 20 20"
-            fill="none"
-          >
-            <path
-              d="M8.33337 5.83334L12.5 10L8.33337 14.1667"
-              stroke="#6C1F1F"
-              stroke-width="1.66667"
-              stroke-linecap="round"
-              stroke-linejoin="round"
-            />
-          </svg>
-        </div>
-      </Button>
-    </div>
+    <button
+      disabled={!props.trigger}
+      className="relative flex h-8 w-28 items-center justify-center rounded-[100px] bg-woh-white text-[#6C1F1F] disabled:text-[#B8BFCB]"
+    >
+      <p className="text-base font-semibold">{t("next")}</p>
+    </button>
```

Comment:
Looks like this component is made from scratch. Would prefer to use `<Button />` component since it already has a built in `isDisabled` class which helps with styling for disabled Button